src/style.css

```
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--main-text-color);
+}
+
+:root {
+  --main-text-color: #000;
+  --active-link-color: #00acdc;
+}
+
+.navbar {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav {
+  line-height: 14px;
+  text-transform: uppercase;
+  color: var(--main-text-color);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.nav__list > li:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: block;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

```suggestion
  display: flex;
  align-items: center;
```
use flex instead of padding